services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js

```diff
@@ -0,0 +1,84 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2024 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+qx.Class.define("osparc.desktop.credits.CreditsNavBarContainer", {
+  extend: qx.ui.core.Widget,
+
+  construct: function() {
+    this.base(arguments);
+
+    this._setLayout(new qx.ui.layout.Grow());
+
+    this.set({
+      zIndex: osparc.utils.Utils.FLOATING_Z_INDEX,
+      maxWidth: this.self().WIDTH,
+      backgroundColor: "background-main-2",
+      padding: 10
+    });
+    osparc.utils.Utils.setIdToWidget(this, "creditsNavBarContainer");
+
+    const layout = new qx.ui.container.Composite(new qx.ui.layout.HBox(10));
+
+    const creditsIndicator = new osparc.desktop.credits.CreditsIndicator();
+    const store = osparc.store.Store.getInstance();
+    store.bind("contextWallet", creditsIndicator, "wallet");
+    layout.add(creditsIndicator, {
+      flex: 1
+    });
+
+    const buttonSize = 26;
+    const billingCenterButton = new qx.ui.form.Button().set({
+      appearance: "form-button-outlined",
+      width: buttonSize,
+      height: buttonSize,
+      alignX: "center",
+      alignY: "middle",
+      center: true,
+      icon: "@FontAwesome5Solid/ellipsis-h/12"
```

Comment:
From my understanding the 3 dot or ellipses should remain consistent. Either horizontal or vertical but not both... We should remain the same for the button so I would suggest changing this to vertical too

Reply:
the actions are different, but ok